### DISCUSSION TITLE
Add policy to a readme

### DISCUSSION THREAD

**Discussion by Dum4G** [iptv-org/community#599] [Ideas]
If we moderately control which apps and services should be presented in that list there should be a policy page explaining our requirements for the quality and distribution

**Comment by BellezaEmporium** [comment]
I suggest adding two lines on the "Apps" sections, that could go as followed :

These applications are either online websites, or applications you can install on your device. 
While we prefer applications that are available on the respective platforms' App Store (for security measures), we also allow developers to add their own project, should it be open-source and the code free of malicious intents.
